### DISCUSSION TITLE
Image Preview toolbar buttons only close the preview, when using unstyled and Tailwind as Passthrough Options

### DISCUSSION THREAD

**Discussion by baracuda69** [primefaces/community#1267] [PrimeReact] [answered]
When clicking on an image to open preview, the preview opens as expected (although the image dimensions are too large to fit on-screen - which can be addressed with custom css).

However, the toolbar (with the rotate, zoom and close icons) does not function;
Any button/icon clicked on that toolbar simply closes the image preview (previewcontainer).

I have tried to add some custom css to adjust the z-index of the previewcontainer, toolbar, but to no avail.

**Comment by melloware** [comment]
If you can create a Stackblitz reproducer showing the issue then I would report it as a bug and include your StackBlitz reproducer...

**Comment by baracuda69** [comment]
https://stackblitz.com/edit/stackblitz-starters-3gzjrf?file=app%2Fpage.tsx [https://stackblitz.com/edit/stackblitz-starters-3gzjrf?file=app%2Fpage.tsx]

Note that stackblitz originally configured next.js v13.x, and the problem I mentioned did not exist;
However, after updating to next.js v14 the problem is as reported.

You can, of course, see all package versions in package.json.

Adam

On 23/02/2024 16:00:18, Melloware ***@***.***> wrote:
If you can create a Stackblitz reproducer showing the issue then I would report it as a bug and include your StackBlitz reproducer...

**Comment by melloware** [comment]
That is so weird yeah the showcase uses Next 12 and doesn;t have the issue.  I am looking at the code and there is not much going on here i would think that would cause this.  I am going to convert this conversation into an issue on GitHub

**Reply by melloware** [reply, marked as answer]
Issue opened: https://github.com/primefaces/primereact/issues/6037
